hack/generate-values.sh

```diff
@@ -24,7 +24,7 @@ if [[ $# -ge 2 ]]; then
   GCP_SERVICE_ACCOUNT_JSON="$2"
 
   if [[ ! -r ${GCP_SERVICE_ACCOUNT_JSON} ]]; then
-    echo "Error: Unable to read GCP service account JSON from file: ${GCP_SERVICE_ACCOUNT_JSON}" >&2
+    >&2 echo "Error: Unable to read GCP service account JSON from file: ${GCP_SERVICE_ACCOUNT_JSON}" >&2
```

Comment:
@cwlbraa Did you mean to have this redirection at both the beginning and end of the line? Is this some bash magic?